.circleci/config.yml

```diff
@@ -290,14 +290,14 @@ jobs:
       - restore_cache:
           keys:
             # prefer the exact match
-            - vm-<< parameters.arch >>_go.mod_{{ checksum "go.sum" }}_{{ checksum "mk/dependencies/deps.lock" }}_{{ checksum ".circleci/config.yml" }}
+            - vm-<< parameters.arch >>_go.mod_{{ checksum "go.sum" }}_{{ checksum "mk/dependencies/deps.lock" }}_{{ checksum ".circleci/config.yml" }}_{{ checksum "mk/dev.mk" }}
```

Comment:
should you just add a hash of mk/dev.mk to https://github.com/kumahq/kuma/blob/master/tools/dev/install-dev-tools.sh#L19 ?